src/api/app/services/notification_service/notifier.rb

```diff
@@ -41,8 +41,7 @@ def create_notification_per_subscription(subscription, channel)
     end
 
     def create_notification?(subscriber, channel)
-      return false if subscriber&.away?
-      return false if channel == :rss && !subscriber&.rss_token
+      return false if subscriber.nil? || subscriber.away? || (channel == :rss && !subscriber.try(:rss_token))
```

Comment:
This is a small fix for a bigger one that should be handled in a different PR. We created an issue for that https://github.com/openSUSE/open-build-service/issues/9958